### PR TITLE
docs(pretrained_models): fix num parameters

### PR DESCRIPTION
**Pull request by amineabdaoui**
# What does this PR do?

This PR corrects the number of parameters of pretrained BERT based models presented in the documentation.

Sometimes the difference between a given model and its pairs is important. For instance:
`bert-base-uncased` has **110M parameters** but `bert-base-multilingual-cased` has more than **178M parameters**, even if both models share the same architecture (12-layers, 768-hidden, 12-heads).

The difference is due to the vocabulary size:
`bert-base-uncased` uses a vocabulary of **30k** entries while `bert-base-multilingual-cased` uses a vocabulary of **119k** entries.

To compute the number of parameters:
``` python
from transformers import AutoModelForMaskedLM

bert_base = AutoModelForMaskedLM.from_pretrained('bert-base-uncased')
print(bert_base.num_parameters())

bert_multiling = AutoModelForMaskedLM.from_pretrained('bert-base-multilingual-cased')
print(bert_multiling.num_parameters())
```

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dimiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests), 
      Pull Request section?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

Particularly: @LysandreJik and @sgugger

### CODE REVIEW

File: docs/source/pretrained_models.rst [hunk 1]
```diff
@@ -11,49 +11,49 @@ For a list that includes community-uploaded models, refer to `https://huggingfac
...
 |                    |                                                            | | Trained on cased English text.                                                                                                      |
 |                    +------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+
 |                    | ``bert-base-multilingual-uncased``                         | | (Original, not recommended) 12-layer, 768-hidden, 12-heads, 110M parameters.                                                        |
-|                    |                                                            | | Trained on lower-cased text in the top 102 languages with the largest Wikipedias                                                    |
+|                    |                                                            | | Trained on lower-cased text in the top 168 languages with the largest Wikipedias                                                    |
```
Comment (LysandreJik) [inline docs/source/pretrained_models.rst:24]: It seems this isn't true, it says 102 in the [original implementation](https://github.com/google-research/bert/blob/master/multilingual.md#models)

Comment (LysandreJik) [reply docs/source/pretrained_models.rst:24]: Ah, did you mean to change the number of parameters instead? It's indeed 168M.

Comment (amineabdaoui) [reply docs/source/pretrained_models.rst]: Thanks !
Corrected : 23702e74418ef30736d3f052dbe4c19cfaae2da6

**Review by LysandreJik — APPROVED** [review]
Great, thanks!

**Pull request merged**